src/html.js

```diff
@@ -1,13 +1,17 @@
 import React from "react";
 import PropTypes from "prop-types";
 import { PORTAL_TARGETS } from "constants/domNodes";
+import { IS_BUILD } from "constants/env";
 
 export default function HTML(props) {
   return (
     <html {...props.htmlAttributes}>
       <head>
         <meta charSet="utf-8" />
         <meta httpEquiv="x-ua-compatible" content="ie=edge" />
+        {IS_BUILD && (
+          <script async src="https://www.google-analytics.com/analytics.js" />
```

Comment:
No idea why the old analytics snippet wasn't loading, because all it does is append this script to head. Since that's all it's doing anyway (the weird `i,s,o,g,r,a,m` snippet), I just added it directly.